distribution/docs/src/main/resources/content/_managing/_running/ddf-service.adoc

```diff
@@ -84,6 +75,25 @@ root${at-symbol}localhost# ln -s ${home_directory}/bin/${branding-lowercase}-ser
 root${at-symbol}localhost# chkconfig ${branding-lowercase}-service --add
 root${at-symbol}localhost# chkconfig ${branding-lowercase}-service on
 ----
+. (Windows only, if the system's `JAVA_HOME` variable has spaces in it) Edit `${home_directory}/etc/${branding-lowercase}-wrapper.conf`.
+Put quotes around `wrapper.java.additional.n` system properties for n from 1 to 13 like so:
```

Comment:
Had to add this workaround due to Karaf bug, which is fixed in Karaf 4.2.2: https://issues.apache.org/jira/browse/KARAF-5908